cupy/core/_backend.pyx

```diff
@@ -0,0 +1,35 @@
+import os
+
+cdef list _reduction_backends = []
+cdef list _routine_backends = []
+
+
+def _get_backend(str s):
+    if s == 'cub':
+        return BACKEND_CUB
+    # if s == 'cutensor':
+    #     return BACKEND_CUTENSOR
+    raise ValueError('Unknown backend: {}'.format(s))
+
+
+def set_reduction_backends(str s):
```

Comment:
How about using list instead of comma-separated string? I think list is better for parameterized testing or benchmarking.

Reply:
Fixed in https://github.com/cupy/cupy/pull/3461/commits/fbf46c5c9d3859114c3cb017872e8d703b203150.